src/plugins/plot/legend/PlotLegendItemExpanded.vue

```diff
@@ -169,6 +175,55 @@ export default {
                 this.formattedMinY = '';
                 this.formattedMaxY = '';
             }
+
+            if (this.domainObject.configuration) {
```

Comment:
Some telemetry endpoints that are added to stacked plots don't have configuration. their properties are saved on the parent (stacked plot). In this case, you need to observe the stacked plot to find legend color changes.

See: https://github.com/nasa/openmct/blob/release/2.0.5/src/plugins/plot/inspector/forms/SeriesForm.vue#L301
and
https://github.com/nasa/openmct/blob/release/2.0.5/src/plugins/plot/inspector/PlotOptionsEdit.vue#L122

Reply:
Thanks Shefali!

That is correct. The domain object in this case is the stacked plot (the parent). So if you drag a telemetry object into the stacked plot, delete it, and re-add it. It should retain the same color properties. 

This check in particular distinguishes domain objects with configurations (stacked plot, overlays, etc.) against telemetry objects which are always blue, non-editable, and do not need a color observer so it will not run the code. 

I hope that makes sense!

Reply:
Non editable objects still need color observers - but changes will be saved to the parent stacked plot. I'll take a look at this and push a change of how this could be done.